vbuild/apkbuild.py

```
@@ -26,6 +34,62 @@ def wrapper(self: "APKBUILD") -> str | None:
+
+            quoted_value += token
+            continue
+
+        offset, name = get_token(value, offset)
+        source = "$" + name
+        if name == "{":
+            offset, name = get_token(value, offset)
+            source += name
+            offset, next_token = get_token(value, offset)
+            source += next_token
+            if next_token != "}":
```

Ruff (0.15.1)

[error] Possible hardcoded password assigned to: "next_token"

(S105)